plugin/storage/grpc/config/config.go

```diff
@@ -41,6 +46,30 @@ type Configuration struct {
 	remoteConn *grpc.ClientConn
 }
 
+type ConfigV2 struct {
+	Configuration `mapstructure:",squash"`
```

Comment:
The idea of cfg v2 is to have a clean break, not to keep using v1 config (ok to use it internally when starting the implementation, but not as external config struct)